plenum/test/monitoring/test_request_time_tracker.py

```diff
@@ -152,7 +152,7 @@ def test_request_tracker_performs_garbage_collection_on_remove_instance():
     req_tracker.remove_instance(0)
     assert digest in req_tracker
 
-    req_tracker.remove_instance(2)
```

Comment:
What is the reason for this change?

Reply:
In the last version "2" was a new index of replica3 in the list. But now a key in the dictionary is instance id.